pkg/assetfetcher/http.go

```diff
@@ -100,12 +100,12 @@ func (c *NexposeAssetFetcher) FetchAssets(ctx context.Context, siteID string) (<
 	pagedErrChan := make(chan error, siteAssetResp.Page.TotalResources)
 	for _, asset := range siteAssetResp.Resources {
 		if !asset.hasBeenScanned() {
-			// no need to continue processing a Nexpose Asset that has never been scanned
-			stater.Count("assetreceived.skipped", 1)
+			stater.Count("assetmissingfield", 1, fmt.Sprintf("site:%s", siteID), "TimeStamp")
```

Comment:
task: Tags should be sent as key/value pairs. It is technically possible and valid to send only a value but it does not have the effect that most folks want from a tag. It's probably easiest to conceive of metrics tags like a SQL column used in a GROUP BY or WHERE. For example, adding the `site:<siteID>` tag enables queries like `SELECT count(*) FROM assetmissingfield GROUP BY site` which would show the distribution of bad assets by site. 

Using a single value tag makes it difficult to write the kinds of queries people expect. A metric system will have to either ignore the single value tag or interpret it in a way that enables using the key/value store. Most systems I've seen convert single value tags in to keys with empty values. This would convert `TimeStamp` into `TimeStamp:`. The result is that it would only enable queries like `SELECT count(*) FROM assetmissingfield GROUP BY TimeStamp` which is not particularly useful. It's more likely that an operator will want to do something like `SELECT count(*) FROM assetmissingfileld GROUP BY field` to see a distribution.